project.mml

```diff
@@ -230,6 +230,7 @@ Layer:
             way,
             "natural",
             waterway,
+            water,
```

Comment:
(Since GitHub lacks general comments for reviews, I attach it to the first change, to make clear what problems that need to be fixed - sorry)

There are two problems I see here:
- dark wood spots (this is exactly the same problem as it was when testing color fading changes) - a blocker for me 
- I have a big question about what problem exactly resolves reducing colors at low zoom? The whole rationale is about fixing problems which will be probably fixed by #3952, it's not clear what this particular solution would change in comparison